src/clients/lan_auto.rb

```diff
@@ -159,7 +160,13 @@ def main
         @ret = deep_copy(@autoyast)
       elsif @func == "Write"
         @progress_orig = Progress.set(false)
-        @ret = Lan.WriteOnly
+
+        @ret = LanUdevAuto.Write
+        Builtins.y2error("Writing udev rules failed") if !@ret
+
+        @ret = Lan.WriteOnly && @ret
```

Comment:
yes. best effort approach. LanUdevAuto only writes udev rules according AY profile. Lan.Write* writes configuration for devices, dns, routing, ...